app/views/home/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class="row">
+<% @photos_low_res.find_each do |photo| %>
+  <div class="col-xs-6 col-md-3">
+    <img src="<%= CloudStorage::Object.new(photo.object_key).url %>" class="img-thumbnail"/>
```

Comment:
This object construction can be moved to a helper, so you may be able to clean your view like this:

```erb
<img src="<%= cloud_storage_object_url(photo) %>" class="img-thumbnail">
```